lib/resources/collection/index.js

```diff
@@ -567,6 +567,10 @@ Collection.prototype.save = function (ctx, fn) {
         }
 
         delete item.id;
+        if ((_.keys(prev).length > 0) && (_.keys(item).length === 0)) {
+          return done(null, {id: id});
```

Comment:
I think this should return an empty object `{}` instead of the `id`. Maybe the id is intended to be hidden.